src/jit/lower.cpp

```diff
@@ -3570,16 +3570,25 @@ void Lowering::InsertPInvokeCallProlog(GenTreeCall* call)
         GenTree* frameAddr =
             new (comp, GT_LCL_VAR_ADDR) GenTreeLclVar(GT_LCL_VAR_ADDR, TYP_BYREF, comp->lvaInlinedPInvokeFrameVar);
 
-        // Insert call to CORINFO_HELP_JIT_PINVOKE_BEGIN
-        GenTree* helperCall =
+// Insert call to CORINFO_HELP_JIT_PINVOKE_BEGIN
+#ifdef FEATURE_READYTORUN_COMPILER
```

Comment:
We do not want to have ifdefs on `FEATURE_READYTORUN_COMPILER` in the JIT. We want to have on JIT binary that works for both JIT and R2R compiler.

If `FEATURE_READYTORUN_COMPILER` is ever defined when building the JIT, it sounds like a bug to me. `FEATURE_READYTORUN_COMPILER` should not be ever defined when building the JIT.

Reply:
There are a couple of FEATURE_READYTORUN_COMPILER specific code that already exists... we should probably revisit this and see why we had them. I'll remove the one i'm adding (not needed anymore)

Reply:
> couple of `FEATURE_READYTORUN_COMPILER` specific code

Ok, I forgot about that. `FEATURE_READYTORUN_COMPILER` is actually unconditionally defined when building the JIT.